src/js/store/WebStore.js

```diff
@@ -0,0 +1,23 @@
+export default class WebStore {
+  #key;
+
+  #cached;
+
+  constructor(key) {
+    this.#key = key;
+    this.#cached = JSON.parse(localStorage.getItem(this.#key)) || [];
+  }
+
+  #cache(data) {
+    this.#cached = [...data];
+  }
+
+  load() {
+    return [...this.#cached] || JSON.parse(localStorage.getItem(this.#key));
```

Comment:
```suggestion
    return [...this.#cached]
```
생성자에서 미리 초기화를 해줘서 뒤의 병합연산자가 의미가 없지 않을까요?
만약 캐시 데이터가 없다고해도 빈 배열이 false가 아니기에.. 필요없다고 생각해요

Reply:
어차피 cached의 디폴트가 빈 배열이니 필요가 없겠군요. 수정하겠습니다!